### ISSUE TITLE
autokey vs autokey-py3

### ISSUE THREAD

**Issue by guettli** [autokey/autokey#48]
Hi,

how does autokey compare to [autokey-py3](https://github.com/autokey-py3/autokey)

If a developer wants to add new features, which project should be used to provide pull requests?

**Comment by josephj11** [comment]
Thanks for your inquiry.

AutoKey is the original version of the software. This version has been
without a developer for some time.

I took over the project to keep it alive. I do a lot of support on our
support list (which is just now beginning to support the -py3 version as
well), but I am a beginner with Python and cannot support the code base
itself. I have also worked on some of the documentation and hope to do more
(with a couple of others) as time allows.

autokey-py3 is a fork of the project with the code converted to Python 3.x,
several enhancements, and various bug fixes.

Troy is the active developer for the autokey-py3 fork. So that would be the
best place for contributions. We would very much appreciate any energy and
code you wish to contribute.

I don't think Troy lurks here or on our support list, so you'll have to
contact him via the autokey-py3 page on GitHub. If you have any difficulty
reaching him, let me know and I will contact him directly.

We look forward to hearing more from you.

Joe

P.S.

Feel free to visit our support list at
https://groups.google.com/forum/#!forum/autokey-users . Currently, we don't
have anyone there fluent in Python or who is familiar with AutoKey
internals.
